lib/ohai/plugins/linux/platform.rb

```diff
@@ -59,6 +59,15 @@ def get_redhatish_version(contents)
       contents = File.read("/etc/parallels-release").chomp
       platform get_redhatish_platform(contents)
       platform_version contents.match(/(\d\.\d\.\d)/)[0]
+    elsif File.exists?('/etc/os-release')
```

Comment:
Do we know if this file will exist on any other platforms? Could break detection for those, if so. Probably safer to put this last.